lib/model-interfaces/langchain/functions/request-handler/adapters/bedrock/claude.py

```diff
@@ -25,23 +24,40 @@ def get_llm(self, model_kwargs={}):
         if "maxTokens" in model_kwargs:
             params["max_tokens_to_sample"] = model_kwargs["maxTokens"]
 
-        return BedrockChat(
-            client=bedrock,
+        return Bedrock(
+            client=client,
```

Comment:
looks like client is not defined above (it's still `bedrock` if you were looking to rename it)

Reply:
I missed the next commit, all good.